### PR TITLE
Add warnings to deprecated CMS profile attributes

### PR DESCRIPTION
**Pull request by hugovk**
PR https://github.com/python-pillow/Pillow/pull/1756 added features to `ImageCMS` and deprecated some others, released in 2016-04-01 on 3.2.0. 

But they're quiet deprecations: only mentioned in docs.

* https://pillow.readthedocs.io/en/stable/reference/ImageCms.html?highlight=deprecated

Changes proposed in this pull request:

* Raise `DeprecationWarning` when a deprecated `ImageCms.CmsProfile` attribute is called
* Update `ImageCms.py` to use non-deprecated attributes internally

### CODE REVIEW

**Comment by hugovk** [comment]
> **codecov/patch** — 72.97% of diff hit (target 84.65%)

I think the patch coverage is below target because of the "Convert tabs to spaces" commit, but overall coverage is fine.

File: src/_imagingcms.c [hunk 1]
```diff
@@ -1005,12 +1015,16 @@ cms_profile_getattr_rendering_intent(CmsProfileObject* self, void* closure)
 static PyObject*
 cms_profile_getattr_pcs(CmsProfileObject* self, void* closure)
 {
+    PyErr_WarnEx(PyExc_DeprecationWarning,
+                 "cms is deprecated. Use padded connection_space instead.", 1);
```
Comment (radarhere) [inline src/_imagingcms.c]:
```suggestion
                 "pcs is deprecated. Use padded connection_space instead.", 1);
```

File: src/PIL/ImageCms.py [hunk 2]
```diff
@@ -819,7 +819,7 @@ def getProfileModel(profile):
         # add an extra newline to preserve pyCMS compatibility
         if not isinstance(profile, ImageCmsProfile):
             profile = ImageCmsProfile(profile)
-        return profile.profile.product_model + "\n"
+        return profile.profile.model + "\n"
```
Comment (radarhere) [inline src/PIL/ImageCms.py]: Is there a reason this doesn't also involve the 'or'?
```suggestion
        return (profile.profile.model or "") + "\n"
```

Comment (hugovk) [reply src/PIL/ImageCms.py]: Sorry for the late reply, I missed this comment. There probably was a reason but I don't remember now! I've added it for consistency and safety.

File: docs/deprecations.rst [hunk 3]
```diff
@@ -79,6 +79,26 @@ PILLOW_VERSION constant
 ``PILLOW_VERSION`` has been deprecated and will be removed in the next
 major release. Use ``__version__`` instead.
 
+ImageCms.CmsProfile attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. deprecated:: 3.2.0
+
+Some attributes in ``ImageCms.CmsProfile`` are deprecated. From 6.0.0, they issue a
```
Comment (radarhere) [inline docs/deprecations.rst]:
```suggestion
Some attributes in ``ImageCms.ImageCmsProfile`` are deprecated. From 6.0.0, they issue a
```

File: docs/deprecations.rst [hunk 4]
```diff
@@ -79,6 +79,26 @@ PILLOW_VERSION constant
 ``PILLOW_VERSION`` has been deprecated and will be removed in the next
 major release. Use ``__version__`` instead.
 
+ImageCms.CmsProfile attributes
```
Comment (radarhere) [inline docs/deprecations.rst]:
```suggestion
ImageCms.ImageCmsProfile attributes
```

File: docs/releasenotes/6.0.0.rst [hunk 5]
```diff
@@ -99,6 +99,24 @@ version.
 
 Use ``PIL.__version__`` instead.
 
+ImageCms.CmsProfile attributes
```
Comment (radarhere) [inline docs/releasenotes/6.0.0.rst]:
```suggestion
ImageCms.ImageCmsProfile attributes
```

File: docs/releasenotes/6.0.0.rst [hunk 6]
```diff
@@ -99,6 +99,24 @@ version.
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some attributes in ``ImageCms.CmsProfile`` have been deprecated since Pillow 3.2.0. From
```
Comment (radarhere) [inline docs/releasenotes/6.0.0.rst]:
```suggestion
Some attributes in ``ImageCms.ImageCmsProfile`` have been deprecated since Pillow 3.2.0. From
```

**Pull request merged**